pybikes/data/compartibike.json

```diff
@@ -0,0 +1,52 @@
+{
+    "instances":[
+       {
+            "tag":"integrabike",
+            "meta":{
+                "latitude":-23.501719,
+                "city":"Sorocaba",
+                "name":"IntegraBike Sorocaba",
```

Comment:
Sorocaba is an implicit part of the name (city). Also, it appears the name is `Integrabike`.